src/config/networkConfig.ts

```diff
@@ -175,100 +218,56 @@ export class NetworkConfig {
     return networkId === 1 || networkId === 4 || networkId === 50
   }
 
-  getConditionalTokensAddress() {
+  getConditionalTokensAddress(): string {
     return networks[this.networkId].contracts.conditionalTokensAddress
   }
 
-  getRealitioAddress() {
+  getRealitioAddress(): string {
     return networks[this.networkId].contracts.realitioAddress
   }
 
-  getTokens() {
+  getTokens(): Token[] {
     return networks[this.networkId].tokens
   }
-}
-
-export const getGraphUris = (networkId: number): { httpUri: string; wsUri: string } => {
-  if (!NetworkConfig.isKnownNetwork(networkId)) {
-    throw new Error(`Unsupported network id: '${networkId}'`)
-  }
 
-  const httpUri = networks[networkId].graphHttpUri
-  const wsUri = networks[networkId].graphWsUri
-  return { httpUri, wsUri }
-}
-
-export const getEarliestBlockToCheck = (networkId: number): number => {
-  if (!NetworkConfig.isKnownNetwork(networkId)) {
-    throw new Error(`Unsupported network id: '${networkId}'`)
+  getGraphUris(): { httpUri: string; wsUri: string } {
+    const httpUri = networks[this.networkId].graphHttpUri
+    const wsUri = networks[this.networkId].graphWsUri
+    return { httpUri, wsUri }
   }
 
-  return networks[networkId].earliestBlockToCheck
-}
-
-interface KnownOracleData {
-  name: string
-  url: string
-  addresses: {
-    [networkId: number]: string
+  getEarliestBlockToCheck(): number {
+    return networks[this.networkId].earliestBlockToCheck
   }
-}
-
-export const knownOracles: { [name in KnownOracle]: KnownOracleData } = {
-  realitio: {
-    name: 'Realitio Team',
-    url: 'https://realit.io/',
-    addresses: {
-      [NetworkIds.MAINNET]: '0x0e414d014a77971f4eaa22ab58e6d84d16ea838e',
-      [NetworkIds.RINKEBY]: '0x576b76eebe6b5411c0ef310e65de9bff8a60130f',
-      [NetworkIds.GANACHE]: '0xDb56f2e9369E0D7bD191099125a3f6C370F8ed15',
-    },
-  },
-  kleros: {
-    name: 'Kleros',
-    url: 'https://kleros.io/',
-    addresses: {
-      [NetworkIds.MAINNET]: '0x0000000000000000000000000000000000000000',
-      [NetworkIds.RINKEBY]: '0x0000000000000000000000000000000000000000',
-      [NetworkIds.GANACHE]: '0x0000000000000000000000000000000000000000',
-    },
-  },
-  unknown: {
-    name: 'Unknown',
-    url: '',
-    addresses: {},
-  },
-}
 
-export const getKnowOracleFromAddress = (networkId: number, address: string): KnownOracle => {
-  for (const key in knownOracles) {
-    const oracleAddress = knownOracles[key as KnownOracle].addresses[networkId]
+  getTokenFromAddress(address: string): Token {
+    const tokens = networks[this.networkId].tokens
 
-    if (!oracleAddress) {
-      continue
+    for (const token of tokens) {
+      const tokenAddress = token.address
+      if (tokenAddress.toLowerCase() === address.toLowerCase()) {
+        return token
+      }
     }
```

Comment:
It is just style, but for me is clearer if you map tokens to his address lower cases and then uses find. But is the same.